brave/brave-core
# [Brave News]: Build feed off the main thread

Resolves https://github.com/brave/brave-browser/issues/35695



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. In a new profile click the `Learn More` link on the Enable Brave News popup
2. Navigate back
3. The page should not crash
4. Enable Brave News
5. The publishers should load
6. The feed should load
7. Disable Brave news
8. Enable Brave news
9. THe publishers should load 
10. THe Feed should load
11. Disable Brave news, refresh the page
12. Enable Brave news
13. The publishers should load
14. The feed should load
